spalp_example/src/EventSubscriber/SpalpExampleConfigLocationAlterSubscriber.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Drupal\spalp_example\EventSubscriber;
+
+use Drupal\spalp\Event\SpalpConfigLocationAlterEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Class SpalpExampleConfigLocationAlterSubscriber.
+ *
+ * @package Drupal\spalp_example\EventSubscriber
+ */
+class SpalpExampleConfigLocationAlterSubscriber implements EventSubscriberInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents() {
+    $events[SpalpConfigLocationAlterEvent::CONFIG_LOCATION_ALTER] = 'doConfigLocationAlter';
+    return $events;
+  }
+
+  /**
+   * Use a custom location for spalp configuration.
+   *
+   * @param \Drupal\spalp\Event\SpalpConfigLocationAlterEvent $event
+   *   Spalp Config Location Alter event.
+   */
+  public function doConfigLocationAlter(SpalpConfigLocationAlterEvent $event) {
+
+    // These are the default locations.
+    // In a real implementation, you would change them.
+    // For instance, your JSON might be in the libraries directory.
+    // TODO: proper dependency injection example.
+    $module_path = drupal_get_path('module', 'spalp_example');
+    $locations = [
+      'config' => $module_path . '/spalp_example.config.json',
+      'schema' => $module_path . '/spalp_example.config.schema.json',
```

Comment:
Should we use APP_ID constant defined in https://github.com/Capgemini/spalp/blob/8.x-1.x/spalp_example/src/SpalpExampleInterface.php#L13 instead of string "spalp_example"

Reply:
maybe, but I think that might make it harder to read

these are the filenames, so best to keep it simple IMO